x-pack/plugins/siem/public/components/timeline/body/renderers/process_draggable.test.tsx

```diff
@@ -0,0 +1,218 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import * as React from 'react';
+import { mountWithIntl } from 'test_utils/enzyme_helpers';
+
+import { TestProviders } from '../../../../mock';
+
+import { isNillOrEmptyString, ProcessDraggable } from '.';
+
+describe('ProcessDraggable', () => {
```

Comment:
i appreciate how this component attempts to surface the best information we have at the time of rendering, and the unit tests that verify it